.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -17,5 +17,6 @@ Closes: #
 
 
 ---
+- [ ] I have considered adding unit tests for this change. If I decided not to add them, I have provided a brief explanation below (optional):
```

Comment:
I appreciated a lean WooCommerce-iOS PR checklist compared to WordPress-iOS one. I learned that adding a new item in a checklist, even well-meaning, doesn't help actually address them 😀.

However, if this PR is meant to be approved and not discussed, let me know.

Reply:
I agree that more items don't necessary lead to everyone following them. However, missing unit tests in particular have been an issue for quite some time within Woo. We believe this might increase awareness as well as make it easier for the reviewer to challenge the PR if it's missing unit tests without an explanation. 

I think we shouldn't leave this particular item on the template for ever, but having it there for 4-6 months might help. Wdyt?

For example, the item about release notes seems has been helping quite a bit AFAICT.

Reply:
Okay, that sounds reasonable. I don't know all the background and it may prove to be helpful 👍